core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -996,7 +996,7 @@ private[ui] object ApiHelper {
     HEADER_EXECUTOR -> TaskIndexNames.EXECUTOR,
     HEADER_HOST -> TaskIndexNames.HOST,
     HEADER_LAUNCH_TIME -> TaskIndexNames.LAUNCH_TIME,
-    HEADER_DURATION -> TaskIndexNames.DURATION,
+    HEADER_DURATION -> TaskIndexNames.EXEC_RUN_TIME,
```

Comment:
Nit: add comment to explain why it should be executor run time here

Reply:
Thanks. Done

Reply:
Not big deal but there are two spaces after `SPARK-26109:`

Reply:
Thanks. Done

Reply:
Done.